front/app/components/ProjectCard/LargeProjectCard.tsx

```diff
@@ -0,0 +1,333 @@
+
+import ProjectHeader, {
+  ContentHeaderBottomMargin,
+  ContentHeaderHeight,
+} from './ProjectHeader';
+import ScreenReaderContent from './ScreenReaderContent';
+import {
+  handleProjectCardOnClick,
+  handleProjectTitleOnClick,
+  ProjectImage,
+  ProjectTitle,
+} from './utils';
+
+const Container = styled(Link)`
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: row;
+  margin-bottom: 25px;
+  position: relative;
+  cursor: pointer;
+  align-items: stretch;
+  justify-content: space-between;
+
+  ${defaultCardStyle};
+
+  ${isRtl`
+    flex-direction: row-reverse;
+  `}
+
+  ${media.desktop`
+    ${defaultCardHoverStyle};
+  `}
+`;
+
+const ProjectImageContainer = styled.div`
+  width: 50%;
+  display: flex;
+  aspect-ratio: ${CARD_IMAGE_ASPECT_RATIO} / 1;
+  margin-right: 10px;
+  overflow: hidden;
+  position: relative;
+  border-top-left-radius: 4px;
+  border-bottom-left-radius: 4px;
+
+  ${media.phone`
+    aspect-ratio: ${CARD_IMAGE_ASPECT_RATIO} / 1;
+  `}
+`;
+
+const ProjectContent = styled.div`
+  flex-grow: 1;
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+  padding-top: 18px;
+  padding-bottom: 35px;
+  padding-left: 68px;
+  padding-right: 32px;
+
+  ${media.tablet`
+    padding-left: 20px;
+    padding-right: 20px;
+  `}
+
+  ${isRtl`
+    align-items: flex-end;
+    padding-right: 68px;
+    padding-left: 32px;
+  `}
+`;
+
+const ContentHeader = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 0px;
+  padding-bottom: ${ContentHeaderBottomMargin}px;
+  border-bottom: solid 1px #e0e0e0;
+
+  &.noContent {
+    ${media.desktop`
+      height: ${ContentHeaderHeight + ContentHeaderBottomMargin}px;
+    `}
+  }
+
+  &.hasRightContent.noLeftContent {
+    justify-content: flex-end;
+  }
+
+  &.hasContent {
+    margin-bottom: ${ContentHeaderBottomMargin}px;
+  }
+
+  ${isRtl`
+    text-align: right;
+  `}
+`;
+
+const ContentBody = styled.div`
+  width: 100%;
+  flex-grow: 1;
+  display: flex;
+  flex-direction: column;
+  padding-top: 20px;
+  max-width: 400px;
+  justify-content: center;
+`;
+
+const ProjectDescription = styled.div`
+  color: ${colors.textSecondary};
+  font-size: ${fontSizes.base}px;
+  line-height: normal;
+  font-weight: 300;
+  overflow-wrap: break-word;
+  word-wrap: break-word;
+  word-break: break-word;
+  margin-top: 15px;
+
+  ${isRtl`
+    text-align: right;
+  `}
+`;
+
+const ContentFooter = styled.div`
+  flex-shrink: 0;
+  flex-grow: 0;
+  flex-basis: 45px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 8px;
+
+  &.hidden {
+    border: none;
+    margin-top: 0px;
+  }
+`;
+
+export interface InputProps {
+  projectId: string;
+  hideDescriptionPreview?: boolean;
+  className?: string;
+  showFollowButton?: boolean;
+}
+
+const LargeProjectCard = memo<InputProps>(
+  ({ projectId, hideDescriptionPreview, className, showFollowButton }) => {
```

Comment:
This part where we fetch all the data is also almost identical between cards, but I guess it's a bit tricker to make reusable